util.py

```diff
@@ -6,10 +6,10 @@ def parse_names(names):
 
 
 def split_name_code(name_ligapokemon):
-    names = name_ligapokemon.split(" (")[0]
+    names = ' '.join(name_ligapokemon.split(" ")[:-1])
     name_pt, name_en = parse_names(names)
 
-    code = name_ligapokemon.split(" (")[1].split("/")[0]
+    code = name_ligapokemon.split(" ")[-1].split("/")[0][1:]
```

Comment:
Split on `" ("` is necessary to ensure that is getting codes
Then, removing unnecessary `[0][1:]` on the final
```suggestion
    code = name_ligapokemon.split(" (")[-1].split("/")[0]
```

Reply:
More clear without using the slicing too